backend/migrations/20210506010327_create_analytics_table.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE book_analytics (
+    id    text PRIMARY KEY,
+    clicks integer[],
```

Comment:
I would add a `NOT NULL DEFAULT array_fill(0, array[366])` here, and remove the `(clicks) / array_fill` from your insert query below. That way we can ensure that the array never gets deleted somehow.